### PR TITLE
Export flows to locust.io (http://locust.io)

### PR DESCRIPTION
**Pull request by zlorb**
Add options for exporting flows to be used with locust load test tool (https://github.com/locustio/locust):
- Export locust:  complete code for running locust test on a specific flow.
- Export locust task: additional code snippet, to be added as an extra flow to a pre-existing locust code (as above).

### CODE REVIEW

File: mitmproxy/flow_export.py [hunk 1]
```diff
@@ -2,6 +2,12 @@
 import netlib.http
 from textwrap import dedent
 
+import sys, re
+if (sys.version_info > (3, 0)):
+    from urllib.parse import urlparse
+else:
+    import urlparse
```
Comment (Kriechi) [inline mitmproxy/flow_export.py]: @mhils started using `six` for py2/3 compatible imports in recent commits to master.
Maybe you could try to adopt it here as well?

**Comment by Kriechi** [comment]
Did you take a look at #976?
Maybe we can try to improve the situation (if possible) and plan for future refactorings...

File: test/mitmproxy/test_flow_export.py [hunk 2]
```diff
@@ -1,361 +1,361 @@
-from textwrap import dedent
```
Comment (Kriechi) [inline test/mitmproxy/test_flow_export.py]: Your branch looks a bit messy with a lot of diff.
Could you please try to rebase to current master and keep the file modes intact (0644 vs. 0755).
Ideally Github should only list very few files in the "Files changed" tab - and not 106!

**Comment by zlorb** [comment]
Github client on Windows seems to mess up file permissions on mounted drives.
Will clean up and create a new pull request.

**Pull request closed without merging**